lib/virtual-fs/src/mem_fs/mod.rs

```diff
@@ -54,6 +54,8 @@ struct ArcFileNode {
     metadata: Metadata,
 }
 
+// TODO: this is broken!!! A `VirtualFile` stores its own offset,
```

Comment:
Should be a `FIXME: ` ;) .

And probably a followup issue.

Reply:
Sure :D